Sprint-3/revise/investigate/find.js

```diff
@@ -20,6 +20,10 @@ console.log(find("code your future", "z"));
 // Pay particular attention to the following:
 
 // a) How the index variable updates during the call to find
+// 1. It updates as it iterates through the str when find() is called and will run / increment at least once while the condition is true or a character match is found.
```

Comment:
The explanations given in 1, 2, 3, 4 are not very clear; some of them seems incorrect (to my understanding).

Can you try using ChatGPT to see how else you can explain the code?